sonar-project.properties

```diff
@@ -5,8 +5,6 @@ sonar.sources=src
 sonar.exclusions=**/node_modules/**,**/*.spec.ts
 sonar.host.url=https://sonarcloud.io
 sonar.verbose=true
-sonar.login=74d7142294b4d7c59c7d6d288b6a985f977fc0bd
-
 
 # sonar.tests=src/app
```

Comment:
can we also remove these commented out properties @carstonsrc

Reply:
^ @carstonsrc please update this file - I think we determined those lines were necessary uncommented